launch/tier4_system_launch/launch/system.launch.xml

```diff
@@ -85,4 +94,16 @@
       </include>
     </group>
   </group>
+
+  <!-- Dummy Diag Publisher -->
+  <group if="$(eval &quot;'$(var run_mode)'=='planning_simulation'&quot;)">
+    <include file="$(find-pkg-share dummy_diag_publisher)/launch/dummy_diag_publisher.launch.xml">
+      <arg name="config_file" value="$(var dummy_diag_publisher_param_path)"/>
+      <arg name="extra_config_file_sensor" value="$(var sensor_launch_pkg)/config/dummy_diag_publisher/sensor_kit.param.yaml"/>
```

Comment:
I think we also need to add this parameter file in [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch).

Reply:
Adding default value (empty param file) to the file above is another way.  Or we need to add an empty param file to [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch)?

Reply:
Both approach is fine for me! (If I had to choose, I would go for the latter one, since setting the default parameter may be confusing)

Reply:
OK, I will add an empty param file to [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch).

Reply:
@kminoda How do we deal with the gap between  [autowarefoundation/autoware_launch](https://github.com/autowarefoundation/autoware_launch) when we modify [tier4_system_launch/launch/system.launch.xml](https://github.com/autowarefoundation/autoware.universe/pull/2840/files/bd5dac0ddf2299504782cac2773e7f990e332e12#diff-985682f001a2c5fdfbf452fbeb3424fba1093af16bdfd0f618ca228117906adc)?  [autowarefoundation/autoware_launch](https://github.com/autowarefoundation/autoware_launch) does not have dummy_diag_publisher launch implementation. Do we also need empty param files here?

Reply:
@asana17 (Sorry, maybe I couldn't get the point) I will leave the decision up to you, as long as planning simulator works fine with awf/autoware and tier4/pilot-auto. I think it's better to first merge this PR ASAP and mitigate those differences later on.

Reply:
@kminoda Modifying [tier4_system_launch/launch/system.launch.xml](https://github.com/autowarefoundation/autoware.universe/pull/2840/files/bd5dac0ddf2299504782cac2773e7f990e332e12#diff-985682f001a2c5fdfbf452fbeb3424fba1093af16bdfd0f618ca228117906adc) also have effects to awf/autoware, because awf/autoware also use this tier4_system_launch/launch/system.launch.xml from awf/autoware_launch. However, awf/autoware_launch does not have a config file corresponding with [here](https://github.com/asana17/autoware.universe/blob/bd5dac0ddf2299504782cac2773e7f990e332e12/launch/tier4_system_launch/launch/system.launch.xml#L101), and we cannot launch the planning simulator (occurs param file path not found error). To deal with this problem, we also need to put an empty param file in awf/autoware_launch.

Reply:
@kminoda Temporarily placing empty param files for dummy_diag_publisher to [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch) and awf/autoware_launch is better way than setting the default params to tier4_system_launch/launch/system.launch.xml, so I will deal with it.

Reply:
@asana17 I see, make sense, thank you :pray: 
I agree with your proposal!

Reply:
@kminoda I made the PRs.  Could you please review them?
https://github.com/autowarefoundation/autoware_launch/pull/206
https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/55

Reply:
@asana17 Thanks!
Could you double-check if planning_simulator works in awf/autoware? `dummy_diag_publisher` fails in my case:
```
[ERROR] [dummy_diag_publisher-20]: process has died [pid 1052617, exit code -6, cmd '/home/minoda/ghq/github.com/autowarefoundation/autoware/install/dummy_diag_publisher/lib/dummy_diag_publisher/dummy_diag_publi
sher --ros-args -r __node:=dummy_diag_publisher -p use_sim_time:=False -p wheel_radius:=0.383 -p wheel_width:=0.235 -p wheel_base:=2.79 -p wheel_tread:=1.64 -p front_overhang:=1.0 -p rear_overhang:=1.1 -p left_o
verhang:=0.128 -p right_overhang:=0.128 -p vehicle_height:=2.5 -p max_steer_angle:=0.7 --params-file /home/minoda/ghq/github.com/autowarefoundation/autoware/install/autoware_launch/share/autoware_launch/config/s
ystem/dummy_diag_publisher/dummy_diag_publisher.param.yaml --params-file /home/minoda/ghq/github.com/autowarefoundation/autoware/install/sample_sensor_kit_launch/share/sample_sensor_kit_launch/config/dummy_diag_
publisher/sensor_kit.param.yaml --params-file /tmp/launch_params_4nsgtg10'].

```

Reply:
@kminoda Thank you for checking. I found out that the empty parameter file causes this error. I wanted to treat no param as an error when I implemented the dummy_diag_publisher code, so I temporarily fixed parameter files to set "dummy_diag_empty" diag.

Reply:
@asana17 Thank you! It seems that awf/autoware works.